src/Open3D/Visualization/Rendering/Filament/FilamentGeometryBuffersBuilder.cpp

```diff
@@ -118,6 +120,10 @@ GeometryBuffersBuilder::Buffers PointCloudBuffersBuilder::ConstructBuffers() {
                                             VertexBuffer::AttributeType::FLOAT4,
                                             ColoredVertex::GetColorOffset(),
                                             sizeof(ColoredVertex))
+                                 .attribute(VertexAttribute::CUSTOM0, 0,
+                                            VertexBuffer::AttributeType::FLOAT4,
+                                            ColoredVertex::GetTangentOffset(),
+                                            sizeof(ColoredVertex))
```

Comment:
I think it might be helpful to have a comment explaining why CUSTOM0 is used instead of TANGENT (or whatever the enum is), given that it wasn't obvious.